src/components/HomeTab/HomeTab.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+import styles from './HomeTab.module.scss';
+
+import Header from '../Header';
+
+import Navigation from '../Navigation';
+
+import Balance from '../Balance';
+
+import Currency from '../Currency';
+
+import Table from '../Table';
+
+export default function HomeTab() {
+  return (
+    <>
+      <Header />
+      <div className={styles.container}>
+        <div className={styles.LeftPartContainer}>
+          <div className={styles.containerTablet}>
+            <Navigation />
+            <Balance />
+          </div>
+          <div className={styles.CurrencyHiddenContainer}>
+            <Currency />
```

Comment:
If `CurrencyTable` will be a more convenient and suitable name for this component?

```suggestion
            <CurrencyTable />
```